README.md

```diff
@@ -18,7 +18,9 @@ This package has 2 methods:
 
 The example of using:
 ```sh
-import untarjs from "@emscripten-forge/untarjs";
+import { initUntarJS } from "@emscripten-forge/untarjs";
+
+const untarjs = await initUntarJS();
```

Comment:
@AnastasiaSliusar note the change in the API here, this allows loading the WASM once.